src/core/storage/index.ts

```diff
@@ -35,7 +35,8 @@ export function persistConfig(config: Configuration): Promise<Configuration> {
         vapidPublicKey: config.vapidPublicKey,
         serviceWorkerPath: config.serviceWorkerPath,
         autoResubscribe: config.autoResubscribe,
-        pushPrompts: config.pushPrompts
+        pushPrompts: config.pushPrompts,
+        tenantId: config.tenantId
```

Comment:
Note this isn't used in the worker, but in future we should be careful if we need to change the persisted config

Reply:
This will we rewritten eventually. Tested a use case when the site is not updated but the worker is, works properly.